ingestion/functions/parsing/USA/USA.py

```diff
@@ -18,6 +19,7 @@
             os.pardir,os.pardir, 'common'))
     import parsing_lib
```

Comment:
I think this stanza (found in all of the parsers) is redundant Lambda-era noise. We create the image ourselves from a Dockerfile now, and we put `common` in the right place, so `common.parsing_lib` should be discoverable both in tests and in production. Should we clean this up, or am I wrong?

Reply:
From the comments, this is about getting pytest to run, but that should be possible by using python -m pytest instead of just pytest, so yes, this should be fine to delete :tada: